RELEASE-NOTES.md

```diff
@@ -1,5 +1,79 @@
+    name: str
+    greeting: str = "Hello"
+
+
+@flow(log_prints=True)
+def my_flow(person: Person, name: str = "world"):
+    print(f'{person.name} says, "{person.greeting}, {name}!"')
+
+
+if __name__ == "__main__":
+    my_flow.serve(
+        "testing-params",
+        enforce_parameter_schema=True,
+    )
+
+```
+
+An attempt to run the created deployment with invalid parameters will fail and give a reason the flow run cannot be created:
+```bash
+> prefect deployment run 'my-flow/testing-params' -p person='{"name": 1}'
+
+Error creating flow run: Validation failed for field 'person.name'. Failure reason: 1 is not of type 'string'
+```
+
+You can enable parameter enforcement via `prefect deploy` with the `--enforce-parameter-schema` flag or by setting `enforce_parameter_schema` to `True` in your `prefect.yaml` file.
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10773
+
+### Enhancements
+- Add API route for work pool counts — https://github.com/PrefectHQ/prefect/pull/10770
+- Add pattern matching for deployment name in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/10772
+- Add CLI command to get default base job template — https://github.com/PrefectHQ/prefect/pull/10776
+
+### Fixes
+- Make paths relative rather than absolute in the `prefect dev build-ui` command — https://github.com/PrefectHQ/prefect/pull/10390
+- Lower the upper bound on pinned pendulum library — https://github.com/PrefectHQ/prefect/pull/10752
+- Fix command handling in `run_shell_script` deployment step on Windows — https://github.com/PrefectHQ/prefect/pull/10719
+- Fix validation on concurrency limits — https://github.com/PrefectHQ/prefect/pull/10790
+- Fix Prefect variable resolution in deployments section of `prefect.yaml` — https://github.com/PrefectHQ/prefect/pull/10783
+
+### Documentation
+- Update UI screenshot for role creation — https://github.com/PrefectHQ/prefect/pull/10732
+- Add `push work pools` tag to push work pools guide to raise visibility — https://github.com/PrefectHQ/prefect/pull/10739
+- Update docs with recent brand changes — https://github.com/PrefectHQ/prefect/pull/10736
+- Update Prefect Cloud quickstart guide to include new features — https://github.com/PrefectHQ/prefect/pull/10742
+- Fix broken diagram in workers tutorial — https://github.com/PrefectHQ/prefect/pull/10762
+- Add screenshots to Artifacts concept page — https://github.com/PrefectHQ/prefect/pull/10748
+- Remove boost from block-based deployments page in documentation and improved visibility of `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/10775
+- Add documentation for work-pool get-default-base-job-template — https://github.com/PrefectHQ/prefect/pull/10784
+- Add references to `enforce_parameter_schema` to docs — https://github.com/PrefectHQ/prefect/pull/10782
+- Add documentation for pattern matching in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/10791
+
+## New Contributors
```

Comment:
```suggestion
## New contributors
```

Reply:
I see we had title case in the example below, so maybe not worth changing going forward, but the docs style guide has second level headings in second case, fwiw.